build.sbt

```diff
@@ -941,24 +941,15 @@ lazy val releaseSettings = Seq(
       case _ => Seq[ReleaseStep]()
     }
   },
-  pomExtra := (
-    <url>http://github.com/getquill/quill</url>
-    <licenses>
-      <license>
-        <name>Apache License 2.0</name>
-        <url>https://raw.githubusercontent.com/getquill/quill/master/LICENSE.txt</url>
-        <distribution>repo</distribution>
-      </license>
-    </licenses>
-    <scm>
-      <url>git@github.com:getquill/quill.git</url>
-      <connection>scm:git:git@github.com:getquill/quill.git</connection>
-    </scm>
-    <developers>
-      <developer>
-        <id>fwbrasil</id>
-        <name>Flavio W. Brasil</name>
-        <url>http://github.com/fwbrasil/</url>
-      </developer>
-    </developers>)
+
+  homepage := Some(url("http://github.com/getquill/quill")),
+  licenses := List(("Apache License 2.0", url("https://raw.githubusercontent.com/getquill/quill/master/LICENSE.txt"))),
```

Comment:
Note that the `distribution` section is missing, it turns out that while this is valid for Maven for Ivy its (see https://ant.apache.org/ivy/history/2.4.0/ivyfile/license.html). Even in SBT's own internal library management code, its not possible to add extra fields to the ivy description (see https://github.com/sbt/librarymanagement/blob/a83f75c02e91afa114c77dae1b5a8f084913b1e4/ivy/src/main/scala/sbt/internal/librarymanagement/Ivy.scala#L316)